spec/service_models/miq_ae_service_service_template_catalog_spec.rb

```diff
@@ -0,0 +1,32 @@
+module MiqAeServiceServiceTemplateCatalogSpec
+  describe MiqAeMethodService::MiqAeServiceServiceTemplateCatalog do
+    context "through an automation method" do
```

Comment:
I meant the whole `context "through an automation method" do` block should be removed.